reader/reader_test.js

```diff
@@ -177,3 +177,30 @@ test("write to a map in a compute", function(){
 
 	QUnit.equal(map.complete, false, "value set");
 });
+
+// TODO: How can we test this? Neither onerror or onunhandledrejection catches
+// test("promise readers throw errors (#70)", function() {
```

Comment:
could we have our own "canjs" way of throwing these errors? Maybe part of `can-util/js/dev`?  This way it can be hooked into (or overridden by a developer).